src/main/java/uk/gov/hmcts/juror/api/moj/service/UserServiceModImpl.java

```diff
@@ -257,4 +259,10 @@ String createUsername(String email) {
     boolean doesUserExistWithEmail(String email) {
         return userRepository.existsByEmail(email.trim());
     }
+
+    private void updateUserApprovalLimit(User user, UpdateUserDto updateUserDto) {
+        if (SecurityUtil.isAdministration() || SecurityUtil.isManager() && SecurityUtil.isCourt()) {
```

Comment:
Would be good to have this as its own method like a private boolean canEditApprovalLimit()

Reply:
updated